src/api/osaamiset.ts

```diff
@@ -27,11 +27,29 @@ export const osaamiset = {
       key,
       combiner,
       async (chunk, signal) => {
-        const result = await client.GET('/api/osaamiset', {
+        // eslint-disable-next-line sonarjs/no-clear-text-protocols
+        const skills = chunk.filter((uri) => !uri.startsWith('http://data.europa.eu/esco/occupation/'));
```

Comment:
Maybe this uri could be in some const as it used in four different places in the project. Most likely the uri wont change, but helps to make sure it is in same way in the whole project.